packages/test/local-server-tests/src/test/documentDirty.spec.ts

```diff
@@ -248,6 +248,9 @@ describe("Document Dirty", () => {
 
                 checkDirtyState("after value set", true, 0);
 
+                // Manually flush the ops before disconnecting
+                (dataObject.context.containerRuntime as IContainerRuntime).flush();
```

Comment:
I wonder if better solution to use process.nextTick() (likely insufficient) or setTimtout(0) with await to ensure we are are testing turn-based implementation.
Maybe other way to say it - if we believe that flush() should not be exposed to data store developers, then likely it should not be exposed to UTs as well, and UT code should mimic actual usage patterns.

Reply:
Good point. The test does not actually test that anymore, as the default is no longer turn-based. This is left here for when we actually do change the default. But I will look into changing this to be more explicit.